param/aplimits.par

```diff
@@ -0,0 +1,14 @@
+typeIerror,r,a,.1,0., 1.,"Upper limit on the probability of a type I error"
```

Comment:
would it be more user friendly to rename `typeIerror` and `typeIIerror` something more meaningful like `prob_false_detection` and  `prob_missed_detection` ?

Reply:
Good idea. Will change.